From b31d63745a0555091d4854b5b35d3b8de68314fd Mon Sep 17 00:00:00 2001
From: Kas User <kas@example.com>
Date: Wed, 26 Jul 2023 07:43:58 +0000
Subject: [PATCH] set AUTOCONNECT_RESET_RETRIES_TIMER to 5s

---
 src/settings/nm-settings-connection.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/settings/nm-settings-connection.c b/src/settings/nm-settings-connection.c
index ccbab80..730a78a 100644
--- a/src/settings/nm-settings-connection.c
+++ b/src/settings/nm-settings-connection.c
@@ -28,7 +28,7 @@
 
 #define AUTOCONNECT_RETRIES_UNSET        -2
 #define AUTOCONNECT_RETRIES_FOREVER      -1
-#define AUTOCONNECT_RESET_RETRIES_TIMER 300
+#define AUTOCONNECT_RESET_RETRIES_TIMER   5
 
 #define _NM_SETTINGS_UPDATE2_FLAG_ALL_PERSIST_MODES ((NMSettingsUpdate2Flags) (  NM_SETTINGS_UPDATE2_FLAG_TO_DISK \
                                                                                | NM_SETTINGS_UPDATE2_FLAG_IN_MEMORY \
