From 72f51f2610b919738ffdc0ec546d1cff299f5e6c Mon Sep 17 00:00:00 2001
From: Kas User <kas@example.com>
Date: Thu, 1 Feb 2024 15:50:51 +0000
Subject: [PATCH] hack for netbird: don't change wt0 device listen port

---
 src/devices/nm-device-wireguard.c | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/src/devices/nm-device-wireguard.c b/src/devices/nm-device-wireguard.c
index c916fa4..215815a 100644
--- a/src/devices/nm-device-wireguard.c
+++ b/src/devices/nm-device-wireguard.c
@@ -1420,8 +1420,12 @@ link_config (NMDeviceWireGuard *self,
 	if (NM_IN_SET (config_mode, LINK_CONFIG_MODE_FULL,
 	                            LINK_CONFIG_MODE_REAPPLY)) {
 
-		wg_lnk.listen_port = nm_setting_wireguard_get_listen_port (s_wg);
-		wg_change_flags |= NM_PLATFORM_WIREGUARD_CHANGE_FLAG_HAS_LISTEN_PORT;
+		if( strcmp(nm_connection_get_id(connection), "wt0") == 0){
+			_LOGI (LOGD_DEVICE, "*** HACK for netbird: Don't change listen port for wt0 device!");
+		} else {
+			wg_lnk.listen_port = nm_setting_wireguard_get_listen_port (s_wg);
+			wg_change_flags |= NM_PLATFORM_WIREGUARD_CHANGE_FLAG_HAS_LISTEN_PORT;
+		}
 
 		wg_lnk.fwmark = priv->auto_default_route_fwmark;
 		wg_change_flags |= NM_PLATFORM_WIREGUARD_CHANGE_FLAG_HAS_FWMARK;
